docs/tutorials/gromacs/spack-gromacs.md

```diff
@@ -106,24 +105,18 @@ contains the terraform needed to deploy your cluster.
 
 ## Deploy the Cluster
 
-Use the following commands to run terraform and deploy your cluster.
+Use below command to deploy your cluster.
 
 ```bash
-terraform -chdir=spack-gromacs/primary init
-terraform -chdir=spack-gromacs/primary apply
+./ghpc deploy spack-gromacs
 ```
 
-The `terraform apply` command will generate a _plan_ that describes the Google
-Cloud resources that will be deployed.
-
-You can review the plan and then start the deployment by typing
-**`yes [enter]`**.
-
-The deployment will take about 30 seconds. There should be regular status updates
-in the terminal.
+You can also use below command to generate a plan that describes the Google Cloud resources that will be deployed.
```

Comment:
I would skip this. If they want to view the plan they can display the plan from the deploy command. Or you could point them to instructions file for more detailed terraform commands, but I think you could skip all together.

Reply:
Removed.